storage/integration_test.go

```diff
@@ -1372,66 +1037,53 @@ func TestIntegration_MultiMessageWriteGRPC(t *testing.T) {
 }
 
 func TestIntegration_MultiChunkWriteGRPC(t *testing.T) {
-	ctx := context.Background()
+	multiTransportTest(skipHTTP("gRPC implementation specific test"), t, func(t *testing.T, ctx context.Context, bucket string, _ string, client *Client) {
```

Comment:
IMO this test should be dual-transport probably. It will also be multi-chunk for HTTP via the resumable upload API, and I don't see a similar test for HTTP currently.

Reply:
Ok! Great, done.